app/controllers/advertisement_controller.rb

```diff
@@ -197,7 +197,7 @@ def select_random_post(category = nil)
     if category.nil?
       category = Category.where(use_for_advertisement: true)
     end
-    Post.undeleted.where(last_activity: (Rails.env.development? ? 365 : 7).days.ago..Time.now)
+    Post.undeleted.where(last_activity: (Rails.env.development? ? 365 : 7).days.ago..Time.zone.now)
```

Comment:
Question (curiosity): What's the `.zone` doing here?

Reply:
...good question, blaming Rubocop. Should be `DateTime.now` really. Will fix.